database/migrations/2020_07_01_193902_add_json_stats_to_guns_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddJsonStatsToGunsTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('guns', function (Blueprint $table) {
+            $table->string('json_stats', 1200);
```

Comment:
```suggestion
            $table->json('json_stats');
```

Reply:
I would say yes to this, but we haven't done this for any of the other JSON columns. Additionally, there are frontend implications for how we save this column. Which one makes it easiest on the frontend? If this needs to be JSON, I will have to change them all.